app/controllers/bucket_jobs_controller.rb

```diff
@@ -3,16 +3,17 @@ class BucketJobsController < ApplicationController
   before_action :set_job, only: [:update, :destroy]
 
   def create
-    job = Job.find(params[:job_id])
-    @bucket.add_job(job.id)
+    @job = Job.find(params[:job_id])
+    @bucket.add_job(@job.id)
     session[:bucket] = @bucket.contents
-    flash[:success] = "You have started an application for #{job.title}. You have started #{pluralize(@bucket.total_jobs, 'job')} applications."
-    redirect_to bucket_path(job) 
+    flash[:success] = "You have started an application for #{@job.title}. You have started #{pluralize(@bucket.total_jobs, 'job')} applications."
+    create_job_application
```

Comment:
@jmejia: here's where we're creating the job application in our database. We haven't implemented yet the functionality to delete the started applications from the database that haven't been completed in a week. Do you see any opportunities to refactor? Is this clear that in the bucketjob controller we create a job application?